contracd/src/daystorage.cpp

```diff
@@ -86,66 +86,68 @@ QList<RpiDataItem> DayStorage::findRpiMatches(QList<QByteArray> const &rpis)
 
 QList<ContactMatch> DayStorage::findDtkMatches(QList<DiagnosisKey> const &dtks)
 {
-    bool result;
-    QFile captures;
-    QString leafname;
```

Comment:
@llewelld likes putting every declaration in the begin of a function. Not sure if he's fine with the change here. (I am)

Reply:
i now also split this into to seperate private methods in order to simplify the logic of the seperate steps

Reply:
@llewelld what do you think about this change?

Reply:
@bionade24 i rebased my changes and skipped the refactoring commits for now so you can accept this merge request